recipe/meta.yaml

```diff
@@ -21,16 +21,16 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=42
```

Comment:
You can remove pinning of setuptools because the latest version will be used. As an experiment, try to build with and without setuptools pinning. The result will be the same - the latest version